src/5_0/project.ts

```diff
@@ -403,7 +163,8 @@ function createBaseProject(
 					fileExists: this.program.fileExists,
 					// @ts-expect-error
 					directoryExists: this.program.directoryExists,
-					realpath: undefined,
+					// @ts-expect-error
+					realpath: this.program.realpath || this.projectService.host.realpath?.bind(this.projectService.host),
```

Comment:
Reset this back to the original implementation, originally set this to undefined while triaging issue but should have been this originally.